spec/workers/send_applications_to_provider_worker_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe SendApplicationsToProviderWorker do
```

Comment:
I question the value of this spec. It exposes the implementation details of the worker without testing either its return value or any side effects

Reply:
thats true, the clockwork spec covers the same ground so this one is pretty much redundant. I'll remove it unless you can think of anything more useful it could test given that all it does is delegate the work to a service class?

Reply:
I think remove it. We also test this whenever we run a spec that invokes the cron

Reply:
it's gone!